opennms-webapp/src/main/java/org/opennms/web/admin/notification/noticeWizard/NotificationWizardServlet.java

```diff
@@ -122,6 +122,7 @@ public void doPost(final HttpServletRequest request, final HttpServletResponse r
          */
         try {
             NotifdConfigFactory.init();
+            NotifdConfigFactory.getInstance().reload();
```

Comment:
is that possible to not call reload here ? (your init reload is removed, and postConstruct did not work properly?)